packages/astro/src/runtime/server/index.ts

```diff
@@ -11,6 +11,9 @@ export { createMetadata } from './metadata.js';
 export { escapeHTML, unescapeHTML } from './escape.js';
 
 const voidElementNames = /^(area|base|br|col|command|embed|hr|img|input|keygen|link|meta|param|source|track|wbr)$/i;
+const htmlBooleanAttributes = /^(allowfullscreen|async|autofocus|autoplay|controls|default|defer|disabled|disablepictureinpicture|disableremoteplayback|formnovalidate|hidden|loop|nomodule|novalidate|open|playsinline|readonly|required|reversed|scoped|seamless|itemscope)$/;
+const htmlEnumAttributes = /^(contenteditable|draggable|spellcheck|value)$/;
+const svgEnumAttributes = /^(autoReverse|externalResourcesRequired|focusable|preserveAlpha)$/;
```

Comment:
Not leaving one here, as SVG _may_ be case sensitive. I actually don’t know if this is so or if browsers respect that behavior.

Reply:
SVG is case sensitive 👍🏻